### PR TITLE
Update data-model-analytics-service.md

### PR DESCRIPTION
**Pull request by SanderHolvoet**
Updated _build & release_ entity sets as specified in [this](https://developercommunity.visualstudio.com/content/problem/887697/azure-devops-analytics-api-builds-and-releases-mis.html) topic.

### CODE REVIEW

**Comment by PRMerger19** [comment]
@SanderHolvoet : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/report/extend-analytics/data-model-analytics-service.md [hunk 1]
```diff
@@ -83,14 +83,10 @@ The following EntitySets are only supported with the **v3.0-preview** API versio
...
 > |Branches | Basic information about branches used in tests or pipelines |  ✔️ |
-> |BuildPipelines| Properties for a build pipeline |  ✔️ |
-> |BuildPipelineTasks | Properties for tasks that are used within a pipeline |  ✔️ |
-> |BuildTaskResults | Merged log of all the stages/steps/jobs/tasks within a specific pipeline execution |   ✔️ |
-> |Builds | Execution information for builds |  ✔️ |
-> |ReleaseEnvironments | Experimental, properties of the release environments |  ✔️ |
-> |ReleasePipelines| Experimental, properties for a release pipeline |  ✔️ |
-> |ReleaseStages |  Experimental, properties for stages defined within a release pipeline |  ✔️ |
-> |Releases | Experimental, execution information for releases  |  ✔️ |
+> |Pipelines| Properties for a build pipeline |  ✔️ |
```
Comment (pradeepna) [inline docs/report/extend-analytics/data-model-analytics-service.md]: Just call this "Properties of a pipeline"

File: docs/report/extend-analytics/data-model-analytics-service.md [hunk 2]
```diff
@@ -83,14 +83,10 @@ The following EntitySets are only supported with the **v3.0-preview** API versio
...
+> |PipelineTasks | Properties for tasks that are used within a pipeline |  ✔️ |
+> |PipelineRunActivityResults | Merged log of all the stages/steps/jobs/tasks within a specific pipeline execution |   ✔️ |
+> |PipelineRuns | Execution information for builds |  ✔️ |
```
Comment (pradeepna) [inline docs/report/extend-analytics/data-model-analytics-service.md]: pipeline runs

**Comment by ktoliver** [comment]
@SanderHolvoet Could you address the comments from @pradeepna as we review the pull request? Thanks.

**Comment by PRMerger19** [comment]
@SanderHolvoet : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by SanderHolvoet** [comment]
> @ktoliver Made the necessary changes.

**Comment by ktoliver** [comment]
Thanks @SanderHolvoet 

Requested changes have been made. Merging per [comment from @pradeepna](https://github.com/MicrosoftDocs/vsts-docs/pull/7046#pullrequestreview-351669632).

**Pull request merged**